tsl/src/chunk.c

```diff
@@ -259,3 +260,92 @@ chunk_invoke_drop_chunks(Oid relid, Datum older_than, Datum older_than_type)
 
 	return num_results;
 }
+
+/*
+ * drop_chunk_replica:
+ *
+ * This function drops a chunk on a specified data node. It then
+ * removes the metadata about the association of the chunk to this
+ * datanode on the access node.
+ */
+Datum
+drop_chunk_replica(PG_FUNCTION_ARGS)
+{
+	const char *node_name = PG_ARGISNULL(0) ? NULL : NameStr(*PG_GETARG_NAME(0));
+	Oid chunk_relid = PG_ARGISNULL(1) ? InvalidOid : PG_GETARG_OID(1);
+	List *data_nodes, *replicas;
+	ForeignServer *server;
+	Chunk *chunk;
+	ChunkDataNode *cdn;
+	char *drop_cmd;
+
+	if (!OidIsValid(chunk_relid))
+		ereport(ERROR,
+				(errcode(ERRCODE_INVALID_PARAMETER_VALUE),
+				 errmsg("Invalid chunk specification %u", chunk_relid)));
```

Comment:
We follow the PostgreSQL error style guide. Specifically in this case: "Use lower case for message wording, including the first letter of a primary error message" https://www.postgresql.org/docs/current/error-style-guide.html

Not sure what is meant by "specification" as this concerns a chunk relid. Also, there was already a check that this relid was `InvalidOid` (i.e, `0`), so no need to print it:

```suggestion
				 errmsg("invalid chunk relation")));
```

Reply:
Changed